### PR TITLE
Pdc client add support for build-image-rtt-tests

### PR DESCRIPTION
**Pull request by bliuredhat**
Add build-image-rtt-tests to support pdc client.

JIRA: PDC-1261

### CODE REVIEW

**Comment by simozhan** [comment]
it is better if more test cases added.

**Comment by bliuredhat** [comment]
@simozhan  and @ycheng-aa ; please review this patch.

File: tests/release/tests.py [hunk 1]
```diff
@@ -56,6 +56,7 @@ def test_list_inactive(self, api):
 
     def test_list_all(self, api):
         api.add_endpoint('releases', 'GET', [])
+
```
Comment (simozhan) [inline tests/release/tests.py]: is this a typo? it seems only added a blank line.

**Comment by simozhan** [comment]
And the file name should be rtt_image_tests.py, not rrt_image_tests.py, right?

File: tests/release/tests.py (hunk 1 above)
Comment (bliuredhat) [inline tests/release/tests.py]: yes, I will update it!

**Comment by bliuredhat** [comment]
@simozhan , I will rename the file with "build_image_rtt_tests.py" and this may be more clear.

**Comment by bliuredhat** [comment]
@ycheng-aa , I got the message that "Only test_result fields can be modified by this call. Trying to change anything else will result in 400 BAD REQUEST response.", so I think just ''test_result" canl be updated or be changed.

File: pdc_client/plugins/rrt_image_tests.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                             group='Filtering')
+        list_parser.set_defaults(func=self.list_build_image_rrt_tests)
+
+        info_parser = self.add_action('info', help='display details of a build image rtt tests')
+        info_parser.add_argument('build_nvr', metavar='BUILD_NVR')
+        info_parser.add_argument('image_format', metavar='IMAGE_FORMAT')
+        info_parser.set_defaults(func=self.build_image_rrt_tests_info)
+
+        update_parser = self.add_action('update', help='Partial update an exist' +
+                                        'build image rtt tests by test_result')
```
Comment (simozhan) [inline pdc_client/plugins/rrt_image_tests.py]: If you think there are too many words in one line, you can 
update_parser = self.add_action('update', help='Partial update an existing \
                                                     build image rtt tests by test_result')

**Comment by simozhan** [comment]
There is no test case for json format output

**Comment by ycheng-aa** [comment]
@bliuredhat it is what i mean, only test_result could be changed. 

BTW you dont' have to push --force every time, do it in the final is fine.

**Comment by bliuredhat** [comment]
@ycheng-aa ，yes, I accepted, thank you~!

**Comment by bliuredhat** [comment]
@simozhan and @ycheng-aa ; please review this patch again~!

**Comment by ycheng-aa** [comment]
Looks good for me.

**Pull request merged**